pbj-core/gradle/modules.properties

```diff
@@ -0,0 +1,3 @@
+com.github.spotbugs.annotations=com.github.spotbugs:spotbugs-annotations
+com.google.protobuf=com.google.protobuf:protobuf-java
+org.antlr.antlr4.runtime=org.antlr:antlr4-runtime
```

Comment:
These are the official names, and hence these move into the plugin: https://github.com/gradlex-org/java-module-dependencies/commit/1a89818e330ae4f7707d1327378cc5d8a14ede3e

Wit the next plugin update, these can be removed.